oaknational/Oak-Web-Application
# fix: plug in new mv LESQ-456

## Description

Music year: {owa_music_year}

- Updates share_mv to version 1

## Issue(s)

Fixes #LESQ-456

## How to test

1. Go to https://deploy-preview-2103--oak-web-application.netlify.thenational.academy
2. Click on any share page, all share video formats should have changed to X min

## Screenshots

How it used to look (delete if n/a):
<img width="740" alt="Screenshot 2023-11-21 at 07 56 58" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/1c7eafd9-dea6-42e6-b3df-c6f363d76e38">


How it should now look:

<img width="1111" alt="Screenshot 2023-11-21 at 07 56 21" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/a3b589ac-d9e5-47dd-b5a6-76f08842e9f8">

## Checklist
